Mobile and desktop version

sentayhu19 opened the pull request:
**in this pull**

- Desktop and Mobile version of the web site
 - Transform added on images 
- Hamburger menus
- about page and index page
- More Button Added on the mobile version
- https://www.loom.com/share/b354f51e724c4a41a1df55dd791b525b
- https://www.loom.com/share/9b82766d69c74cf48d418a22508a659c

sentayhu19 committed:
mobile version desgin
ham burger menu , our blog section, and blogers list secion

sentayhu19 committed:
desktop version of index and about page and linters error fixed

sentayhu19 committed:
Es linters err fixed

sentayhu19 committed:
resizing issue fixed with our blogs section

sentayhu19 committed:
transform added on images and padding fixed

sentayhu19 committed:
alignment on about page heading an styling and heading title text cliping

sentayhu19 committed:
Readme file created

sentayhu19 committed:
linters error fixed with background-clip

sentayhu19 committed:
header icons changed navigation active added and also alignment

sentayhu19 committed:
margines added for blogs section and with for signup btn

sentayhu19 committed:
stylelint error fixed

sentayhu19 committed:
line break for designed by

line break for designed by

sentayhu19 committed:
screenshot added

sentayhu19 committed:
screenshot link added

screenshot link added

Kingstalux commented on about.html:
```diff
@@ -0,0 +1,117 @@
+    <header class="header">
+        <div class="mobile-hidden social-media-top">
+            <div ><img src="./assets/images/facebook.png" class="social-media-top-item1" alt="facebook" ></div>
+          <div>  <img src="./assets/images/Twitter.png" class="social-media-top-item1" alt="twitter" ></div>
```
- [ ] Great job so far 👍 It would be great to add a white Facebook and Twitter icon so as to follow the design more closely and also remove the extra YouTube icon.
- Your navbar:
![Annotation 2022-03-10 193220](https://user-images.githubusercontent.com/85113567/157732402-2582d89f-212b-4b93-a21c-1fc32ff7a2c1.jpg)
- Required design:
![Annotation 2022-03-10 193243](https://user-images.githubusercontent.com/85113567/157732444-a3f02b0b-df61-40c2-a0c3-6292ea1f0faa.jpg)

Kingstalux commented on about.html:
```diff
@@ -0,0 +1,117 @@
+        <div class="about-main-section-item3"><p>If you need help brainstorming, use a worksheet, online tools, or Google Docs to keep track of possible ideas and topics. Do word association or mind map exercises. Talk to friends and colleagues
+             (or folks on the interwebs) for ideas, or browse blogs in your niche to get a sense for what’s already been done. Use prompts. Dump all of your ideas into a single place and always carry around a notebook (or keep a virtual one, like a SwipeFile).</p>
+        </div>
+        <div class="about-main-section-item4 contact">
+            <p class="about-join"> If you want to be Blogger on our Site or if you have questions about our Blog app contact us through our E-mail</p>
+            <p class="about-email">cesblogs@gmail.com</p>
+        </div>
+    </div>
+            </section>
+            <section class="about-us">
+<h2 class="about-title">About our Blog web site</h2>
```
- [ ] Nice design on this page 💯 It would be better to add some more margin to your `About our blog website` section since its too close to the horizontal line just as in the required design
- Your design:
![Annotation 2022-03-10 193340](https://user-images.githubusercontent.com/85113567/157732921-f8cf4962-e313-4856-a5ef-b6f005562982.jpg)
- Required design:

![Annotation 2022-03-10 193413](https://user-images.githubusercontent.com/85113567/157732974-4a57fb77-44d5-40ca-a034-aa8976f6dce7.jpg)

Kingstalux commented on about.html:
```diff
@@ -0,0 +1,117 @@
+                            <div><img src="./assets/images/logo.png" alt="logo" class="logo"></div>
+                            <div><p class="about-rights">we are Bloggers and your life will be good with our site <br><br> &copy; 2022 CES Blogs All rights reserved</p></div>
+                                    </div>
+                    </footer>
```
- [ ] You'll notice that in the mobile version, the bottom of the `About` page contains a partner's section and a white `footer` section. It would be great if you could implement that as well 👍 
- Your design:
![Annotation 2022-03-10 194705](https://user-images.githubusercontent.com/85113567/157733578-d08dabe2-24d4-4580-b2d3-4eccc24452ab.jpg)
- Required design:
![Annotation 2022-03-10 193745](https://user-images.githubusercontent.com/85113567/157733607-27afc874-6a3c-4c87-8a57-aee7959c6902.jpg)

Kingstalux commented on index.html:
```diff
@@ -0,0 +1,81 @@
+    <section>
+        <h3>Our Blogers</h3>
+        <div id="our-blogers"></div>
+    </section>
```
- [ ] Good job generating the speaker's section dynamically from JavaScript but you'll notice that in the mobile version, only 2 speakers should be displayed together with a `more` button.  When a user clicks on the `more` button the rest of the speakers will then be displayed. Kindly implement this functionality 👍 
- Required design:
![Annotation 2022-03-10 193647](https://user-images.githubusercontent.com/85113567/157734740-bf4dff79-cf58-4c36-a1b1-53b2e60af664.jpg)

Kingstalux reviewed (changes requested):
Hi @sentayhu19 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
![68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f5754646735474252343558364e6278714a4b2f67697068792e676966](https://user-images.githubusercontent.com/85113567/157734900-5fb6e156-994b-45cb-912d-0c53b31440ec.gif)



## Suggested changes
_Check the comments under the review._
- [ ] Your video presentation did not contain your mobile version and a piece of interesting code you had when you worked on this project. Kindly record another video with all the necessary information 😄 
- [ ] Your page breaks at exactly `768px` and your `Start blogging` button is out of place right up to `1140px`. It would be great to implement your mobile version from around `992px` to `1024px` to prevent this kind of behavior. Kindly rectify this issue.
- Your page:
![Annotation 2022-03-10 193033](https://user-images.githubusercontent.com/85113567/157735583-fcb2c016-c39a-4565-94e3-97b8523a78dc.jpg)


 
![Annotation 2022-03-10 193053](https://user-images.githubusercontent.com/85113567/157735552-40cd334a-1b3d-4d3e-b48e-3131d9563961.jpg)

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

sentayhu19 committed:
read more added, margines fixed,color fixed
page breking at 768 fixed, white color on social media , about page margines, mobile version footer containes partners, and secc more button aded on the mobile version

sentayhu19 committed:
Merge branch 'mobile-and-desktop-version' of https://github.com/sentayhu19/capstone-project into mobile-and-desktop-version

sentayhu19 committed:
top position of main-section-item3 fixed

sentayhu19 committed:
linters error fixed

sentayhu19 committed:
margines fixed on nav

sentayhu19 committed:
padding fixed

sentayhu19 committed:
linters error fixed

sentayhu19 committed:
padding and margin on about page fixed

thecodechaser commented on about.html:
```diff
@@ -0,0 +1,126 @@
+                 <nav class="navigation">
+        <div id="cover">
+            <img id="close" class="dektop-hidden" src="./assets/images/close.png" alt="close window">
+            <ul id="menu" class="dektop-hidden">
+                <li ><a  class="menu-item" href="./index.html">Home</a></li>
+                <li ><a  class="menu-item" href="#">Our Blog</a></li>
+                <li ><a  class="menu-item" href="#">Join</a></li>
+                <li ><a  class="menu-item" href="#">Sponsor</a></li>
+            </ul>
```
- [ ] When I open the mobile menu and switch to the desktop screen the mobile menu is still on the screen it should disappear. Please make sure when we switch the screen from mobile to desktop the mobile menu should disappear.
 - You can use media queries for that if the screen cross the min-width you can set your mobile menu `display: none`

See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157938553-491d3f46-4be0-4d28-9b88-209a6573cee5.png)

thecodechaser commented on about.html:
```diff
@@ -0,0 +1,126 @@
+</div>
+            </section>
+
+            <div class="partners dektop-hidden">
+                <h3 class="partners-title">Partner</h3>
+                <ul class="parnter-group1">
+                <li><img src="./assets/images/mozilla.svg" class="parnter1" alt="mozilla"></li>
+                <li><img src="./assets/images/google.svg" class="parnter1"  alt="google"></li>
+                <li> <h2 class="parnter2">NAVER</h2></li>
+                <li> <h2 class="parnter2 mobile-hidden">daumkakao</h2></li>
+                <li> <h2 class="parnter2 fa-brands fa-airbnb mobile-hidden">Samsung</h2></li>
+                </ul>
+                <ul class="parnter-group2">
+                    <li> <h2 class="parnter3 dektop-hidden">daumkakao</h2></li>
+                    <li> <h2 class="parnter3 dektop-hidden">Samsung</h2></li>
+                </ul>
+                </div>
```
- [ ] Please make sure the partner's section is in the center and follow the Behance design

### Your design

![image](https://user-images.githubusercontent.com/83803414/157940293-b47cdb73-8fdd-4a2e-9d24-d2ef748552ce.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/157940483-02eb2641-ddbf-4d87-82d4-87ee6c0d2b0c.png)

thecodechaser reviewed (approved):
## STATUS: APPROVED 🟢

## Hi @sentayhu19👋, 

### You did a marvelous job on the capstone project and this is one of the best projects I have seen so far!

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- Good presentation video. ✔️ 

## Optional suggestions
- [ ]  I am marking these changes as optional because they were missed by the previous reviewer, But I strongly suggest considering them. Please check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sentayhu19 committed:
fixed mobile menu showing on desktop
fixed mobile menu showing on dedsktop and partners section centered
